### DISCUSSION TITLE
Converting from version 1

### DISCUSSION THREAD

**Discussion by smithart** [cpvalente/ontime#554] [Q&A]
I've been a user of ontime from the beginning, and would really like to upgrade, but I can't seem to find any information on migrating my configuration from version 1 to the newer versions.

**Comment by cpvalente** [comment]
Hi @smithart, thank you for your question.
When v2 came out the project files were completely compatible. I believe this to be the case still

In that case, all you would have to do is to install v2 and import your project file
If, for some reason, that is not working for you. Reach out at mail@getontime.no and I will convert the file for you
